docs/resources/Channel.md

```diff
@@ -580,6 +588,17 @@ Edit a previously sent message. You can only edit messages that have been sent b
 | content | string | the new message contents (up to 2000 characters) |
 | embed | [embed](#DOCS_RESOURCES_CHANNEL/embed-object) object | embedded `rich` content |
 
+## Suppress Embed(s) on a Message % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/messages/{message.id#DOCS_RESOURCES_CHANNEL/message-object}/suppress-embeds
```

Comment:
The title of this endpoint doesn't feel quite right with others; it's too verbose. Not that it is a rule or anything, but I think endpoint titles should be able to be easily interpreted into function names in languages; in the case of suggestion, `suppress_message_embeds(..)`.
```suggestion
## Suppress Message Embeds % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/messages/{message.id#DOCS_RESOURCES_CHANNEL/message-object}/suppress-embeds
```

Reply:
On it

Reply:
Thanks!